src/components/Card/Card.module.scss

```
@@ -0,0 +1,37 @@
+@import 'styles/styles.scss';
+
+.card {
+  font-size: 0;
+
+  &:hover {
+    box-shadow: 0 4px 10px 0px rgba(0, 0, 0, 0.25);
+  }
+}
+
+.img {
+  width: 100%;
+  height: auto;
+}
+
+.content {
+  background-color: #ffffff;
```

```suggestion
  background-color: $background-elements;
```

и в Button тоже можно ее использовать